<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Restaurante</class>
 <widget class="QDialog" name="Restaurante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 6pt;
background-color: rgb(170, 170, 127);

border: 1px solid black; 
border-style: solid;
border-radius: 4px; 
border-width: 1px;
border-color: #339;

padding: 3px;
padding-left: 5px;
padding-right: 5px;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nome do  restaurante:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_restaurante">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUrlCharactersOnly</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Valor pago pela empresa:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_valor">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_adicionar">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_remover">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Remover</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_editar">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Editar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_cancelar">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_Voltar">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_salvar">
          <property name="styleSheet">
           <string notr="true">font-size: 8pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QListWidget" name="listWidget_lista">
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollBar:vertical {
      width: 40px;
  }</string>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_restaurante</tabstop>
  <tabstop>lineEdit_valor</tabstop>
  <tabstop>pushButton_adicionar</tabstop>
  <tabstop>pushButton_remover</tabstop>
  <tabstop>pushButton_editar</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
  <tabstop>listWidget_lista</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
